ci/cloudbuild/dockerfiles/demo-opensuse-leap.Dockerfile

```diff
@@ -50,6 +49,103 @@ ENV PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:/usr/local/lib64/pkgconfig
 ENV PATH=/usr/local/bin:${PATH}
 # ```
 
+# #### Abseil
```

Comment:
I would continue to pull this from the packages.

Reply:
Done. Thanks.